docs/source/usage/deployment.rst

```diff
@@ -0,0 +1,118 @@
+==========
+Deployment
+==========
+
+This guide uses Zoom as our service, your service may be different, be sure to name it something appropriately. 
+
+***************
+Getting Started
+***************
+
+  1. Clone the master branch of the repository at “https://github.com/mozilla/mozdef-event-framework” to the local file system.
+  2. Before any action, decide on the various parameter and environment variable values, as these are referenced throughout the CF templates and serverless framework configuration.
+     The following are the environment variables with example values:
+
+     * PROJECT: MozDef-EF
+     * SERVICE: zoom
+     * STAGE: dev
+     * API_PATH: /
```

Comment:
API_PATH should not be `/`, but some other string. If `/` is specified here, the URL for the API would end in double slash `//`, which is not what we want.

Maybe we can change it to `zoom` or simply `api`.

Reply:
yeah I didn't know what this should be, so just added a / for now.
I'll craft an example path

Reply:
done